manifests/init.pp

```diff
@@ -1292,6 +1339,33 @@
           }
         }
         '5.11': {
+          file { 'pam_d_cron':
```

Comment:
Curious why these files were not already in the module since the module already supposedly supports Solaris 11